staging/src/k8s.io/api/networking/v1/types.go

```diff
@@ -426,7 +426,7 @@ type HTTPIngressPath struct {
 	//   the IngressClass. Implementations can treat this as a separate PathType
 	//   or treat it identically to Prefix or Exact path types.
 	// Implementations are required to support all path types.
-	PathType *PathType `json:"pathType,omitempty" protobuf:"bytes,3,opt,name=pathType"`
+	PathType *PathType `json:"pathType" protobuf:"bytes,3,opt,name=pathType"`
```

Comment:
If we're removing omitempty shouldn't this type also change to a pointer? I'm not sure if that's backwards compatible though.